### PR TITLE
bump cpal to 0.15

### PR DESCRIPTION
**Pull request by 21pages**
#3762 
wouldn't help unless the device format is not F32, U16, I16

### CODE REVIEW

**Comment by rustdesk** [comment]
👍

File: src/client.rs [hunk 1]
```diff
@@ -801,9 +801,17 @@ impl AudioHandler {
...
             cpal::SampleFormat::I16 => self.build_output_stream::<i16>(&config, &device)?,
+            cpal::SampleFormat::I32 => self.build_output_stream::<i32>(&config, &device)?,
+            cpal::SampleFormat::I64 => self.build_output_stream::<i64>(&config, &device)?,
+            cpal::SampleFormat::U8 => self.build_output_stream::<u8>(&config, &device)?,
             cpal::SampleFormat::U16 => self.build_output_stream::<u16>(&config, &device)?,
+            cpal::SampleFormat::U32 => self.build_output_stream::<u32>(&config, &device)?,
+            cpal::SampleFormat::U64 => self.build_output_stream::<u64>(&config, &device)?,
+            cpal::SampleFormat::F32 => self.build_output_stream::<f32>(&config, &device)?,
+            cpal::SampleFormat::F64 => self.build_output_stream::<f64>(&config, &device)?,
+            f => bail!("unsupported audio format: {:?}", f),
```
Comment (rustdesk) [inline src/client.rs:814]: can remove this f to support all format?

Comment (rustdesk) [reply src/client.rs:814]: and how about more than 2 channels mentioned in the issue. we can convert to 2 channels.

Comment (21pages) [reply src/client.rs:814]: can't remove f

**Pull request merged**